lib/common/windowController.py

```diff
@@ -173,11 +187,24 @@ def move_window_pos(self, pos_x=None, pos_y=None, window_width=None, window_heig
 
         # Try to move window
         if self.window_type == 'linux2':
-            return self.wmctrl_move_window()
+            return self.move_window_linux()
         elif self.window_type == 'win32':
-            return self.pywin32_move_window()
+            return self.move_window_win32()
         elif self.window_type == 'darwin':
-            return self.appscript_move_window()
+            return self.move_window_mac()
         else:
             logger.warning('Doesn\'t support moving window [{}] on platform [{}].'.format(self.window_name_list, self.window_type))
             return False
+
+    def close_window(self):
+        # Try to move window
+        if self.window_type == 'linux2':
+            return self.close_window_linux()
```

Comment:
in my opinion, the 'window' is not 'Windows' or 'win32' :P
So, I prefer the way 1.